### ISSUE TITLE
Selecting with a boolean (more generally, with tpl-padded attributes) corrupts output.

### ISSUE THREAD

**Issue by lmwnshn** [cmu-db/terrier#1028]
Issue found and fixed by @tanujnay112 in [this commit](https://github.com/cmu-db/terrier/pull/994/commits/c288ea4f7947ac5ce789426cc97c6b3f047ca4ff).

TPL Context adds padding for structs [here](https://github.com/cmu-db/terrier/blob/master/src/execution/ast/context.cpp#L285). This was not accounted for in PostgresPacketWriter's [WriteDataRow](https://github.com/cmu-db/terrier/blob/master/src/network/postgres/postgres_packet_writer.cpp#L262).

Before (main):
```
terrier=# select proisagg, proisagg, proisagg, prooid from pg_proc;
 proisagg | proisagg | proisagg | prooid 
----------+----------+----------+--------
 t        | f        |          |      0
 t        | f        |          |      0
 t        | f        |          |      0
 t        | f        |          |      0
 t        | f        |          |      0
 t        | f        |          |      0
 t        | f        |          |      0
 t        | f        |          |      0
 f        | f        |          |      0
 f        | f        |          |      0
 f        | f        |          |      0
 f        | f        |          |      0
 t        | f        |          |      0
 f        | f        |          |      0
(14 rows)
```

After (TPL2):
```
terrier=# select proisagg, proisagg, proisagg, prooid from pg_proc;
 proisagg | proisagg | proisagg | prooid 
----------+----------+----------+--------
 t        | t        | t        |     84
 t        | t        | t        |     85
 t        | t        | t        |     86
 t        | t        | t        |     87
 t        | t        | t        |     88
 t        | t        | t        |     89
 t        | t        | t        |     90
 t        | t        | t        |     91
 f        | f        | f        |     94
 f        | f        | f        |     95
 f        | f        | f        |     96
 f        | f        | f        |     97
 t        | t        | t        |     92
 f        | f        | f        |    150
(14 rows)
```

**Comment by lmwnshn** [comment]
@mbutrovich Could you take a look at the fixing commit too? Thanks!

**Comment by mbutrovich** [comment]
Fix looks good to me.

**Comment by apavlo** [comment]
I love the SQL boolean type...

**Comment by lmwnshn** [comment]
Fixed by #994.
